WordPress/src/main/java/org/wordpress/android/viewmodel/history/HistoryViewModel.kt

```diff
@@ -98,7 +115,40 @@ class HistoryViewModel @Inject constructor(
             items.add(HistoryListItem.Footer(footer))
         }
 
-        _revisions.value = items
+        return items
+    }
+
+    private fun fetchRevisionAuthorDetails(authorsId: List<String>) {
+        PeopleUtils.fetchRevisionAuthorsDetails(site, authorsId, object : FetchUsersCallback {
+            override fun onSuccess(peopleList: List<Person>, isEndOfList: Boolean) {
+                val existingRevisions = _revisions.value ?: return
+                val updatedRevisions = mutableListOf<HistoryListItem>()
+
+                existingRevisions.forEach { it ->
+                    var mutableRevision = it
+
+                    if (mutableRevision is HistoryListItem.Revision) {
+                        // we shouldn't directly update items in MutableLiveData, as they will be updated downstream
+                        // and DiffUtil will not catch this change
+                        mutableRevision = mutableRevision.copy()
+
+                        val person = peopleList.firstOrNull { it.personID.toString() == mutableRevision.postAuthorId }
+                        if (person != null) {
+                            mutableRevision.authorAvatarURL = person.avatarUrl
+                            mutableRevision.authorDisplayName = person.displayName
+                        }
+                    }
+
+                    updatedRevisions.add(mutableRevision)
+                }
+
+                _revisions.postValue(updatedRevisions)
+            }
+
+            override fun onError() {
+                AppLog.e(AppLog.T.POSTS, "Can't fetch details of post revision authors ")
```

Comment:
```suggestion
                AppLog.e(T.API, "Can't fetch details of revision authors")
```

We can remove the `AppLog` qualification from `AppLog.T.POSTS` since `org.wordpress.android.util.AppLog.T` is an import statement.

Since `HistoryViewModel` is used for both pages and posts, we should remove the `post` part of the error message and update the `POSTS` log type to `API`.